examples/step-8/doc/intro.dox

```diff
@@ -12,26 +12,69 @@ understood as well.
 
 @dealiiVideoLecture{19}
 
-In this tutorial program we will want to solve the elastic equations. They are
-an extension to Laplace's equation with a vector-valued solution that
+In this tutorial program we will want to solve the 
+<a href="https://en.wikipedia.org/wiki/Linear_elasticity">elastic equations</a>.
+They are an extension to Laplace's equation with a vector-valued solution that
 describes the displacement in each space direction of a rigid body
 which is subject to a force. Of course, the force is also
 vector-valued, meaning that in each point it has a direction and an
-absolute value. The elastic equations are the following:
+absolute value. 
+
+One can write the elasticity equations in a number of ways. The one that shows
+the symmetry with the Laplace equation in the most obvious way is to write it
+as
+@f[
+  -
+  \text{div}\,
+  (C \nabla \mathbf{u})
+  =
+  \mathbf f,
+@f]
+where $\mathbf u$ is the vector-valued displacement at each point,
+$\mathbf f$ the force, and $C$ is a rank-4 tensor (i.e., it has four
+indices) that encodes the stress-strain relationship -- in essence,
+it represents the 
+<a href="https://en.wikipedia.org/wiki/Hooke%27s_law">"spring constant"</a> in
+Hookes law that relates the displacement to the forces. $C$ will, in many
+cases, depend on $\mathbf x$ if the body whose deformation we want to
+simulate is composed of different materials.
+
+While the form of the equations above is correct, it is not the way
+they are usually derived. In truth, the gradient of the displacement
+$\nabla\mathbf u$ (a matrix) has no physical reality whereas its
+symmetrized version,
+@f[
+\varepsilon(\mathbf u)_{kl} =\frac{1}{2}(\partial_k u_l + \partial_l u_k),
+@f]
+does and is typically called the "strain". (Here and in the following,
+$\partial_k=\frac{\partial}{\partial x_k}$. We will also use the
+<a href="https://en.wikipedia.org/wiki/Einstein_notation">Einstein summation
+convention</a> that whenever the same index appears twice in an equation,
+summation over this index is implied; we will, however, not distinguish
+between upper and lower indices.)
+With this definition of the strain, the elasticity equations
+then read as
+@f[
+  -
+  \text{div}\,
+  (C \varepsilon(\mathbf u))
+  =
+  \mathbf f,
+@f]
+which you can think of as the more natural generalization of the Laplace
+equation to vector-valued problems. (The form shown first is equivalent to
+this form because the tensor $C$ has certain symmetries, namely that
```

Comment:
```suggestion
this form because the tensor $\mathsf{C}$ has certain symmetries, namely that
```
I'm not sure what is the right convention for fourth rank tensors when you don't write the indices. Wikipedia uses $\mathsf{C}$
https://en.wikipedia.org/wiki/Linear_elasticity
I've also seen in some papers $\mathbf{C}$
You can apply this modification in the rest of the patch.